src/adhocracy_frontend/checkcode_and_compile.cfg

```diff
@@ -135,7 +135,7 @@ html_check =
 
 [nodejs]
 recipe = gp.recipe.node
-version = 4.2.4
+version = 4.4.5
 scripts = node npm
```

Comment:
wedriver suggests 4.2 or 5.4, why did you update to 4.4?
https://www.npmjs.com/package/selenium-webdriver